cmd/deploy_test.go

```diff
@@ -222,7 +222,11 @@ func TestAutoSufixWithDnsPrefixInApiModel(t *testing.T) {
 
 		client: &armhelpers.MockACSEngineClient{},
 	}
-	autofillApimodel(deployCmd)
+
+	err = autofillApimodel(deployCmd)
+	if err != nil {
+		t.Fatalf("unexpected error autofilling the example apimodel: %s", err)
```

Comment:
This is the last line of defense? (i.e., no return err?)

Reply:
it's a unit test

Reply:
/me takes off beer goggles